spec/classes/agent_spec.rb

```diff
@@ -160,4 +161,76 @@
       it { should contain_file('/etc/zabbix/zabbix_agentd.conf').with_content %r{^TLSPSKFile=/etc/zabbix/keys/tlspskfile.key$} }
     end
   end
+
+  context 'On Ubuntu' do
+    context 'trusty' do
+      let :facts do
```

Comment:
this would allow us to shorten the overhead. but then we need am itchy case block to mock the systemd fact depending on the OS :(